src/main/java/org/benetech/servicenet/adapter/linkforcare/model/LinkForCareData.java

```diff
@@ -0,0 +1,384 @@
+package org.benetech.servicenet.adapter.linkforcare.model;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+
+@Data
+public class LinkForCareData {
+
+    // ----------
+    // Organization
+    // ----------
+    @SerializedName("ID")
+    private String organizationId;
+
+    @SerializedName("PROVIDER__C")
+    private String organizationName;
+
+    @SerializedName("CREATEDDATE")
+    private String organizationCreatedDate;
+
+    @SerializedName("LASTMODIFIEDDATE")
+    private String organizationModifiedDate;
+
+    @SerializedName("DESCRIPTION__C")
+    private String organizationDescription;
+
+    @SerializedName("WEBSITE__C")
+    private String organizationWebsite;
+
+    @SerializedName("PUBLIC_EMAIL_ADDRESS__C")
+    private String organizationMainEmailAddress;
+
+    // COVID 19 protocols
+    @SerializedName("CURRENT_SERVICE_STATUS_DETAILS__C")
+    private String covid19Protocols;
+
+    @SerializedName("CURRENT_SERVICE_STATUS_FORMULA__C")
+    private String covid19ProtocolsFomula;
+
+    // Phone Number
+    @SerializedName("PUBLIC_PHONE_NUMBER__C")
+    private String organizationPhoneNumber;
+
+    @SerializedName("TOLL_FREE_NUMBER__C")
+    private String organizationPhoneNumberTollFree;
+
+    // ----------
+    // Location
+    // ----------
+    @SerializedName("ADDRESS_LINE_1__C")
```

Comment:
Could location, service, opening hours (and other target relations) be extracted into separate models and be inherited?